### DISCUSSION TITLE
Report User Button

### DISCUSSION THREAD

**Discussion by bobbyiliev** [dailydotdev/daily#1263] [Feature Requests]
Besides reporting posts and comments, users should be able to report other users as well.

**Comment by idoshamun** [comment]
Hi @bobbyiliev,

You are most definitely right. I'll forward it internally :)

**Reply by bobbyiliev** [reply]
Here is one such user that I just noticed posting multiple spam comments:
> https://app.daily.dev/emiliafilipowicz/replies

**Reply by idoshamun** [reply]
Already deleted this profile :)

**Reply by bobbyiliev** [reply]
Hey @idoshamun 

One more such user that is spamming a lot: https://app.daily.dev/hilae

<img width="309" alt="image" src="https://github.com/user-attachments/assets/f23d65ec-6294-4c1e-a2e1-37219acd0e6e">

**Reply by idoshamun** [reply]
We added a system to prevent this spam. It's been wild in the past few days, hopefully, we are able to mitigate it now
